Content/LeagueSandbox-Scripts/Buffs/EzrealWBuff/EzrealWBuff.cs

```diff
@@ -14,30 +14,28 @@ namespace EzrealWBuff
 {
     class EzrealWBuff : IBuffGameScript
     {
-        private StatsModifier _statMod;
-        private IBuff _visualBuff;
+        public BuffType BuffType => BuffType.COMBAT_ENCHANCER;
+        public BuffAddType BuffAddType => BuffAddType.REPLACE_EXISTING;
+        public int MaxStacks => 1;
+        public bool IsHidden => false;
 
-        public void OnActivate(IObjAiBase unit, ISpell ownerSpell)
-        {
-            _statMod = new StatsModifier();
-
-            _statMod.AttackSpeed.PercentBonus = _statMod.AttackSpeed.PercentBonus + (0.15f + 0.05f * ownerSpell.Level);
+        public IStatsModifier StatsModifier { get; private set; } = new StatsModifier();
 
-            unit.AddStatModifier(_statMod);
+        public void OnActivate(IObjAiBase unit, IBuff buff, ISpell ownerSpell)
+        {
+            StatsModifier.AttackSpeed.PercentBonus = StatsModifier.AttackSpeed.PercentBonus + (0.15f + 0.05f * ownerSpell.Level);
 
-            _visualBuff = AddBuffHudVisual("EzrealEssenceFluxBuff", 5f, 1, BuffType.COMBAT_ENCHANCER, unit);
```

Comment:
Visual buffs got deleted with this PR?

Reply:
Not deleted, merged, the packet that was used for Visual Buffs is the just AddBuff packet in this PR.